.github/workflows/integration-test.yaml

```diff
@@ -19,14 +19,16 @@ jobs:
           juju-channel: 3.1/stable
           provider: microk8s
           channel: 1.27-strict/stable
-          microk8s-addons: "hostpath-storage dns"
+          microk8s-addons: "hostpath-storage dns metallb:10.1.1.1-10.1.1.254"
       - name: Enable Multus addon
         continue-on-error: true
         run: |
           sudo microk8s addons repo add community https://github.com/canonical/microk8s-community-addons --reference feat/strict-fix-multus
           sudo microk8s enable multus
           sudo microk8s kubectl -n kube-system rollout status daemonset/kube-multus-ds
           sudo microk8s kubectl auth can-i create network-attachment-definitions
+      - name: Setup upterm session
+        uses: lhotari/action-upterm@v1
```

Comment:
Are we going to merge these debug lines? I know it could be useful, that's why I'm just asking :)

Reply:
No, we can't merge this, because it blocks the actual execution of the tests :)
It will be removed once all problems are figured out.